src/main/java/frc/robot/lib/states/Claw.java

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.lib.states;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+
+import frc.robot.RobotMap;
+
+/** Add your docs here. */
+public class Claw {
+
+    // TODO: positions
+    double closeCubePosition = 0.0;
```

Comment:
Constants like this should be of the `SCREAMING_SNAKE_CASE` format. We should start enforcing programming [naming conventions](https://www.pluralsight.com/blog/software-development/programming-naming-conventions-explained#:~:text=Screaming%20Snake%20Case,-Snek%3A%20IT'S%20A&text=SCREAMING_SNAKE_CASE%20is%20a%20variable%20naming,C%2C%20Javascript%2C%20and%20Java.) for all our variables. All constants should be defined in a singular location in our code base so that we don't have to go hunting for them.